### PR TITLE
Initial keymap creation

### PR DESCRIPTION
**Pull request by masoncowen**
## Description
Initial keymap.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/planck/keymaps/masoncowen/readme.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# The Default Planck Layout
```
Comment (drashna) [inline keyboards/planck/keymaps/masoncowen/readme.md:1]: This file can/should be removed

File: keyboards/planck/keymaps/masoncowen/rules.mk [hunk 2]
```diff
@@ -0,0 +1 @@
+SRC += muse.c
```
Comment (drashna) [inline keyboards/planck/keymaps/masoncowen/rules.mk:1]: This file can/should be removed

File: keyboards/planck/keymaps/masoncowen/keymap.c [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+};
+
+enum planck_keycodes {
+  ZWFP = SAFE_RANGE
+};
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+[_ZWFP] = LAYOUT_planck_grid(
```
Comment (drashna) [inline keyboards/planck/keymaps/masoncowen/keymap.c:31]:
```suggestion
const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {

  LAYOUT_planck_grid(
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/planck/keymaps/masoncowen/config.h [hunk 4]
```diff
@@ -0,0 +1,52 @@
+/* Copyright %YEAR% %YOUR_NAME%
```
Comment (fauxpark) [inline keyboards/planck/keymaps/masoncowen/config.h:1]: Please fill this out.

File: keyboards/planck/keymaps/masoncowen/config.h [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+   - MIDI notes can be added to the keymap
+   - Octave shift and transpose
+   - Virtual sustain, portamento, and modulation wheel
+   - etc.
+*/
+//#define MIDI_ADVANCED
+
+/* override number of MIDI tone keycodes (each octave adds 12 keycodes and allocates 12 bytes) */
+//#define MIDI_TONE_KEYCODE_OCTAVES 2
+
```
Comment (fauxpark) [inline keyboards/planck/keymaps/masoncowen/config.h:49]:
```suggestion
```

File: keyboards/planck/keymaps/masoncowen/keymap.c [hunk 6]
```diff
@@ -0,0 +1,59 @@
+/* Copyright 2015-2017 Jack Humbert
```
Comment (fauxpark) [inline keyboards/planck/keymaps/masoncowen/keymap.c:1]: Should update this as well.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by masoncowen** [comment]
Did a git rebase trying to get rid of the merge commits, should I copy my changes to a fresh branch of master and make a new pull request?

**Comment by drashna** [comment]
> Did a git rebase trying to get rid of the merge commits, should I copy my changes to a fresh branch of master and make a new pull request?

Looks like the rebase messed up?  Not sure why.   I've rebased the PR for you, and that has corrected the issue.

File: keyboards/planck/keymaps/masoncowen/config.h [hunk 7]
```diff
@@ -0,0 +1,17 @@
+/* Copyright 2021 masoncowen
```
Comment (fauxpark) [inline keyboards/planck/keymaps/masoncowen/config.h:1]: This file can be deleted.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by masoncowen** [comment]
Sorry it happened again.
I had to pull from my own repo, I assume for the rebase drashna did.
I entered git rebase (no repo after rebase) to try get rid of the merge commit.
Not sure what I'm meant to do there, I'll just leave the merge commits from now on.

**Comment by drashna** [comment]
> Sorry it happened again.
> I had to pull from my own repo, I assume for the rebase drashna did.
> I entered git rebase (no repo after rebase) to try get rid of the merge commit.
> Not sure what I'm meant to do there, I'll just leave the merge commits from now on.

Are you using an app (such as github desktop) to help manage the repo?  if so, it can cause issues with syncing.

**Comment by masoncowen** [comment]
> > Sorry it happened again.
> > I had to pull from my own repo, I assume for the rebase drashna did.
> > I entered git rebase (no repo after rebase) to try get rid of the merge commit.
> > Not sure what I'm meant to do there, I'll just leave the merge commits from now on.
> 
> Are you using an app (such as github desktop) to help manage the repo? if so, it can cause issues with syncing.

No, purely using git on the command line.

**Review by fauxpark — APPROVED** [review]

**Pull request merged**